accounts/abi/bind/backends/simulated.go

```diff
@@ -675,6 +675,40 @@ func (b *SimulatedBackend) SendTransaction(ctx context.Context, tx *types.Transa
 	return nil
 }
 
+// a replica of above `SendTransaction` function
+func (b *SimulatedBackend) SendTransactionConditional(ctx context.Context, tx *types.Transaction, knownAccounts map[string]map[common.Address]interface{}) error {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+
+	// Get the last block
+	block, err := b.blockByHash(ctx, b.pendingBlock.ParentHash())
+	if err != nil {
+		return fmt.Errorf("could not fetch parent")
```

Comment:
Could you use errors as predefined values? Also, I believe it's useful to add come context to each error and use `errors.New()` for defining new errors and `fmt.Errorf("%w %v - any placeholders", errMyError, var1, var2...)` providing additional context to an error.

Reply:
Right, this can be done.
This function is the replica of the `SendTransaction` function defined just above it. It is not used anywhere, and I have just kept it here because it was included in the interface.